package.json

```diff
@@ -33,6 +33,7 @@
     "dagre": "^0.8.5",
     "downshift": "^6.1.2",
     "gh-pages": "^3.2.3",
+    "html-react-parser": "^3.0.4",
```

Comment:
Is this necessary instead of using `dangerouslySetInnerHTML`?

Reply:
No it is not. Would you recommend using `dangerouslySetInnerHTML` over the parser?

Reply:
If it works without issues, yes. I see no reason to 
- include a new dependency
- parse the HTML that we already expect to be well-formed